README.md

```diff
@@ -2,84 +2,36 @@
-
-## Requirements
-
-In order to run the API locally, you'll need the following installed on your machine:
-
-- [Java Open SDK 11](https://jdk.java.net/archive/)
-- [Maven](https://maven.apache.org/download.cgi)
-- [Git](https://git-scm.com/downloads)
-- [MongoDB](https://www.mongodb.com) although this can be run within a Docker image.
+The `sic-code-api` is an Spring Boot API for searching SIC Codes (industrial classification of a company) which are stored on a Mongo database. The search uses MongoDB full-text search capabilities and can be used to search for a particular SIC Code or text contained in either the SIC Code description or Economic activity. The results are returned in order of relevance determined by the full text-search. In addition there is a filter to only return exact phrase matches.
 
-## Getting Started
-
-Have a Mongo database with the `sic_code` database. See [Data load README](DATALOAD-README.md).
-
-1. Run `make clean build`
-2. Run `./start.sh`
-
-## Running in docker
-
-This project uses the maven jib plugin to build docker images.
-
-For internal Companies House development use, this docker build will be used within Docker CHS Development.
-
-## Environment Variables
-
-The following is a list of mandatory environment variables for the service to run:
+Within Spring Boot, Spring Data Mongo is used to simplify data access to the required Mongo collection (`combined_sic_activities`).
 
-Name                                 | Description                                                               | Example Value
------------------------------------- | ------------------------------------------------------------------------- | ------------------------
-SIC_CODE_API_PORT                    | Application Port                                                          | 8080
-SIC_CODE_API_MONGO_URL               | URL to MongoDB                                                            | `mongodb://mongo:27017`
-SIC_CODE_API_DATABASE                | Name of Sic Code Mongo database                                           | sic_code
+This has a Rest API with the following endpoints:
 
-## What is MongoDB full text search and why do we used it over a regular expression find
+URL                                    | VERB | Notes
+-------------------------------------- | ---- | --------------------------------------------------------------------------
+`/internal/sic-code-search`            | POST | This does a keyword search to search for SIC Codes (with an option to search on one or all of the supplied words)
```

Comment:
I think maybe we should change this. When I initially read it I think it can come across that we could enter 3 words into the search box and only search on one of them if we wanted which doesn't really make sense. 
Perhaps we should just change this to highlight the  "exact match" search, "word match" search and that actual sic codes can be searched also.
What do you think?

Reply:
Updated readme files to hopefully make this clearer (messages in commits)